packages/attune/src/bin/attune/cmd/apt/pkg/add.rs

```diff
@@ -208,6 +208,7 @@ pub async fn run(ctx: Config, command: PkgAddCommand) -> ExitCode {
     }
 }
 
+#[instrument(level = Level::DEBUG)]
 async fn add_package(
```

Comment:
I recommend against gating `tracing::instrument` on level- most tools that interact with tracing spans assume _all_ spans are `info` and there's not a strong reason to do otherwise IME.

What's the reason you changed this?

Reply:
TIL. I was just worried it would be noisy.